web/templates/user_profile_modal.hbs

```diff
@@ -113,13 +113,15 @@
                     <div class="tabcontent" id="manage-profile-tab"></div>
                 </div>
             </main>
-            <footer class="modal__footer">
-                <button type="button" class="modal__btn dialog_cancel_button" aria-label="{{t 'Close this dialog window' }}" data-micromodal-close>{{t "Cancel" }}</button>
-                <button type="button" class="modal__btn dialog_submit_button">
-                    <span>{{t "Save changes"}}</span>
-                    <div class="modal__spinner"></div>
-                </button>
-            </footer>
+            <div id="manage-profile-tab-footer">
```

Comment:
Is it necessary to use an `id` here? Would `class` not suffice?

Reply:
Changed it.